src/main/kotlin/no/skatteetaten/aurora/boober/service/DeployService.kt

```diff
@@ -248,5 +252,17 @@ class DeployService(
 
         return changed
     }
+
+    private fun createImageStream(openShiftResponses: List<OpenShiftResponse>): ImageStream? {
```

Comment:
Ville valgt et annet navn på funksjonen siden vi ikke oppretter noe.

Reply:
findImageStreamResponse() ?

Reply:
Denne oppretter en instans av ImageStream basert på json innholdet. Det kommer ikke helt tydelig frem av ObjectMapper apiet. Jeg hadde disse to create metodene i en factory klasse først, men fjernet det for syns kanskje ikke helt det var verdt en egen spring bean.

Du har rett i at den også søker igjennom listen for å finne den riktige det OpenShiftResponse objektet, men jeg føler ikke findImageStreamResponse heller passer fordi den returnerer en annen type.

Gode navn er ikke alltid så lett å finne :)

Reply:
Jeg er med det.

Reply:
Tja, jeg er ikke enig i at man ikke oppretter noe. For det blir gjort en new ImageStream her, i jackson. Man oppretter et ImageStream som har dataene til OpenShiftResponse.
Det jeg syns er at den kan være litt misvisende pga find, så man kunne eventuelt kalt det noe med convertToImageStream eller noe lignende.